### PR TITLE
luci-app-openvpn getPid() returns always nil

### PR DESCRIPTION
**Pull request by orgixmh**
getPid function can't return pid of openvpn process, instead of pid returns nil.

### CODE REVIEW

File: applications/luci-app-openvpn/luasrc/model/cbi/openvpn.lua [hunk 1]
```diff
@@ -26,7 +26,7 @@ uci:foreach( "openvpn_recipes", "openvpn_recipe",
 )
 
 function s.getPID(section) -- Universal function which returns valid pid # or nil
-	local pid = sys.exec("%s | grep -w '[o]penvpn(%s)'" % { psstring, section })
+	local pid = sys.exec("%s | grep -w '[o]penvpn(%s)' | awk '{print $1}'" % { psstring, section })
 	if pid and #pid > 0 then
 		return tonumber(pid:match("^%d+"))
```
Comment (yousong) [inline applications/luci-app-openvpn/luasrc/model/cbi/openvpn.lua]: Should be okay if we just return `tonumber(pid)`

```
➜  ~ lua
Lua 5.1.5  Copyright (C) 1994-2012 Lua.org, PUC-Rio
> print(tonumber("  123"))
123
> print(tonumber("123"))
123
> print(tonumber(""))
nil
> print(tonumber("nil"))
nil
>
```

Comment (yousong) [inline applications/luci-app-openvpn/luasrc/model/cbi/openvpn.lua]: I suggest changing the pattern to just `%d+`, without the beginning of line char `^`

**Comment by orgixmh** [comment]
@yousong Indeed. Thanks for the tip.

**Comment by hnyman** [comment]
You should squash the commit together, and at the same time properly sign-off the commit.

https://github.com/openwrt/luci/blob/master/CONTRIBUTING.md#advice-on-pull-requests

**Comment by hnyman** [comment]
Likely fixed by https://github.com/openwrt/luci/commit/9f327a3d020c9bd03c3a4a005c4de87353a0c354 so I close this.

**Pull request closed without merging**